test/typescript/actionCreators.ts

```diff
@@ -10,12 +10,73 @@ interface AddTodoAction extends Action {
   text: string
 }
 
-const addTodo: ActionCreator<AddTodoAction> = (text: string) => ({
+// test strict typing of action creators
+// no arguments
+const addTodoNoArgs: ActionCreator<AddTodoAction, []> = () => ({
```

Comment:
an example of declaring an action creator that should be called with no parameters